### ISSUE TITLE
'Failed ports' message is missing flavor

### ISSUE THREAD

**Issue by yurivict** [freebsd/poudriere#593]
In this log the first line has flavors, but the second one doesn't:
```
[00:02:33] Built ports: archivers/py-python-lzo@py36 biology/py-bx-python@py36
[00:02:33] Failed ports: biology/py-bx-python:build-depends
```

**Comment by bdrewery** [comment]
Is 3.6 default for you or 2.7? It may be related to #542

**Comment by yurivict** [comment]
2.7 is default.

**Comment by yurivict** [comment]
Request was: biology/py-bx-python@py27 biology/py-bx-python@py36

**Comment by bdrewery** [comment]
This seems fixed for 3.2.5:
```
[00:01:10] Built ports: misc/freebsd-release-manifests devel/py-setuptools ports-mgmt/poudriere-devel
[00:01:10] Failed ports: devel/py-six@py36:build
```
